includes/controls/groups/base.php

```diff
@@ -543,7 +543,7 @@ private function start_popover( Controls_Stack $element ) {
 			$control_params = array_replace_recursive( $control_params, $popover_options['settings'] );
 		}
 
-		foreach ( [ 'condition', 'conditions' ] as $key ) {
+		foreach ( [ 'global', 'condition', 'conditions' ] as $key ) {
```

Comment:
Can it be removed?

Reply:
I don't remember, should i remove it?

Reply:
We will try after merge with `global-style` panel.